components/worktemplates/components/modal_body_with_illustration.tsx

```diff
@@ -1,26 +1,42 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import React, {useEffect, useRef} from 'react';
+
 import styled from 'styled-components';
 
 export interface ModalBodyWithIllustrationProps {
     className?: string;
-    illustration: string;
+    illustration: [string, string];
```

Comment:
Using an object would help making more clear what's the expected data here rather than indexes `{type: string; src: string}`. Or even having individual props?